src/plot.jl

```diff
@@ -332,10 +333,36 @@ is set to `true` than `PyPlot.jl` is used
 Returns an object that can be used for further plot updates.
 
 """
-plotmap(m::OpenStreetMapX.MapData;roadwayStyle = OpenStreetMapXPlot.LAYER_STANDARD, 
-	width::Integer=600, height::Integer=600, km::Bool=false, use_plain_pyplot::Bool=false) = 
-	plotmap(m.nodes, OpenStreetMapX.ENU(m.bounds), roadways=m.roadways,roadwayStyle = roadwayStyle, width=width, height=height, km=km, use_plain_pyplot=use_plain_pyplot)
+function plotmap(m::OpenStreetMapX.MapData;roadwayStyle = OpenStreetMapXPlot.LAYER_STANDARD, 
+	         width::Union{Integer,Nothing}=nothing, height::Union{Integer,Nothing}=nothing, km::Bool=false, use_plain_pyplot::Bool=false)
+     
+    # Set plot aspect ratio to that of bounds (in meters), unless both height and width are specified
+    if width==nothing || height==nothing
+        # Compute aspect ratio
+        if true
+            p1 = ENU(LLA(m.bounds.min_y, m.bounds.min_x), m.bounds)
+            p2 = ENU(LLA(m.bounds.max_y, m.bounds.max_x), m.bounds)
+            w = abs(p2.east - p1.east)
+            h = abs(p2.north - p1.north)
+            aspect_ratio = h/w
+        else
+            # Why doesn't this work?
```

Comment:
Please clean the code.
Please provide this as a new implementation of `aspect_ratio` (line 42)
Note that aspect ratio is `w/h`, NOT `h/w` - see https://en.wikipedia.org/wiki/Aspect_ratio_(image)
Hence let's use the correct definition